### ISSUE TITLE
React Native Debugger not working with React Native 0.66

### ISSUE THREAD

**Issue by TheFe91** [jhen0409/react-native-debugger#656]
React Native Debugger app version: 0.12.1
React Native version: 0.66
Platform: iOS
Is real device of platform: Yes
Operating System: iOS 15.0.1

I just upgraded my react native project to v0.66, using XCode 13 and debugging on my real iPhone XR, iOS 15.0.1
Before upgrading everything worked nice, now I'm not able to connect to the React Native Debugger (getting errors in console that are NOT related to the app). I followed all your guides but nothing changed
<img width="1680" alt="Schermata 2021-10-05 alle 17 29 47" src="https://user-images.githubusercontent.com/21155677/136054438-a68b34a4-1fc5-4c74-9ec0-ee18180ce8d1.png">

**Comment by TheFe91** [comment]
@efstathiosntonas yes, I did, but downgrading it didn't work, since the debugger still saw the wrong version.
I can add that creating a brand new project with `npx react-native init MyProject` works perfectly even with v0.66, so there could be maybe some conflict in my old project

**Comment by afilp** [comment]
I have the same problem but with upgrading to 4.22.0 globally (npm -g), but it shows the same error message. Seems not to read the package from the global path?
